rpcs3/rpcs3qt/skylander_dialog.cpp

```diff
@@ -162,6 +221,60 @@ const std::map<const u16, const std::string> list_skylanders = {
     {3013, "Grim Creeper"},
     {3014, "Rip Tide"},
     {3015, "Punk Shock"},
+	{3220, "Jet Stream"},
+	{3221, "Tomb Buggy"},
+	{3222, "Reef Ripper"},
+	{3223, "Burn Cycle"},
+	{3224, "Hot Streak"},
+	{3225, "Shark Tank"},
+	{3226, "Thump Truck"},
+	{3227, "Crypt Crusher"},
+	{3228, "Stealth Stinger"},
+	{3231, "Dive Bomber"},
+	{3232, "Sky Slicer"},
+	//{3233, "Clown Cruiser"},
+	{3234, "Gold Rusher"},
+	{3235, "Shield Striker"},
+	{3236, "Sun Runner"},
+	{3237, "Sea Shadow"},
+	{3238, "Splatter Splasher"},
+	{3239, "Soda Skimmer"},
+	//{3240, "Barrel Blaster"},
+	{3241, "Buzz Wing"},
+	{3400, "Fiesta"},
+	{3401, "High Volt"},
+	{3406, "Splat"},
+	{3411, "Smash Hit"},
+	{3412, "Spitfire"},
+	{3413, "Hurricane Jet Vac"},
+	{3414, "Double Dare Trigger Happy"},
+	{3415, "Super Shot Stealth Elf"},
+	{3416, "Shark Shooter Terrafin"},
+	{3417, "Bone Bash Roller Brawl"},
+	{3420, "Big Bubble Pop Fizz"},
+	{3421, "Lava Lance Eruptor"},
+	{3422, "Deep Dive Gill Grunt"},
+	//{3423, "Turbo Charge Donkey Kong"},
+	//{3424, "Hammer Slam Bowser"},
+	{3425, "Dive-Clops"},
+	{3426, "Astroblast"},
+	{3427, "Nightfall"},
+	{3428, "Thrillipede"},
+	{3500, "Sky Trophy"},
+	{3501, "Land Trophy"},
+	{3502, "Sea Trophy"},
+	{3503, "Kaos Trophy"},
+	{210, "Magic Traps"},
```

Comment:
Why are the following entries not correctly sorted in ?

Reply:
Lemme fix that... also I don't know why the indents are weird, I'll also try to fix that.